sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -1512,3 +1512,20 @@ async def hack_mgmt_execute_async(self, operation, op_type, message, timeout=0):
         finally:
             # must reset the mgmt execute method, otherwise other test cases would use the hacked execute method, leading to timeout error
             uamqp.async_ops.mgmt_operation_async.MgmtOperationAsync.execute_async = original_execute_method
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', dead_lettering_on_message_expiration=True)
```

Comment:
I'm wondering if we could use the `CachedServiceBusQueuePreparer` here to save some time in preparing the resources.

in general I would try to use as much  `cached` as possible cuz our livetest running is very long.

and don't forget to complete the message at the end of this test cases if we're using cache, otherwise it might break other cached tests.

but if this one test case has to use a fresh service bus queue, then let's just keep it

Reply:
Sticking to ServiceBusPreparer here since there's an assertion on the message body